query.go

```diff
@@ -301,6 +301,7 @@ func (c *Client) encodeBlock(ctx context.Context, tableName string, input []prot
 	// Note: only blocks are compressed.
 	// See "Compressible" method of server or client code for reference.
 	if c.compression == proto.CompressionEnabled {
+		// TODO SUPPORT COMPRESS
```

Comment:
TODO

Reply:
would you accept this pr or https://github.com/ClickHouse/ch-go/pull/413?

i think this pr is more simple, and col.Str cost most of memory, it is good enough.

Reply:
I've only recently started looking over this repo, still figuring out how it's governed etc